homework_2/src/developer.js

```diff
@@ -0,0 +1,10 @@
+export class Developer {
+  constructor(props) {
```

Comment:
```suggestion
  constructor(props) {
```
я бы опасался давать аргументам название `props`, так как при чтении кода можно случайно перепутать `Developer`, подумав что он не просто класс, а компонент:)